spark/spark1-shims/src/main/scala/org/apache/zeppelin/spark/Spark1Shims.java

```diff
@@ -56,6 +56,10 @@ public String showDataFrame(Object obj, int maxResult) {
     if (obj instanceof DataFrame) {
       DataFrame df = (DataFrame) obj;
       String[] columns = df.columns();
+      // DDL will empty DataFrame
```

Comment:
is row count doesnt matter here?

Reply:
I mean what if row count == 0?

Reply:
row count doesn't matter. Because if column count is 0, then row count must be 0